CHANGELOG.md

```diff
@@ -33,6 +33,8 @@ All notable changes to this project will be documented in this file.
 * ### ALL
     * #### Added
     * #### Changed
+        * Fix [#1811](https://github.com/ni/nimi-python/issues/1811): nimi-python uses scipy which keeps us from adding support to Python 3.10
```

Comment:
Examples are referenced in released documentation but aren't themselves part of the release, so I think it's ok to leave out of the CHANGELOG.md

You are welcome to leave it in though.

Reply:
Reverted.